hyprpm/src/main.cpp

```diff
@@ -114,7 +114,10 @@ int                        main(int argc, char** argv, char** envp) {
 
             auto ret2 = g_pPluginManager->ensurePluginsLoadState();
 
-            if (ret2 != LOADSTATE_OK)
+            if (ret2 == LOADSTATE_HYPRLAND_UPDATED)
+                g_pPluginManager->notify(ICON_INFO, 0, 10000, "[hyprpm] Updated plugins, but Hyprland was updated. Please restart Hyprland.");
+
+            if (ret2 != LOADSTATE_OK || ret2 == LOADSTATE_HYPRLAND_UPDATED)
```

Comment:
why the ||ret2 == LOADSTATE_HYPRLAND_UPDATED  it's redundant lol

Reply:
oop right